astroid/__init__.py

```diff
@@ -82,6 +82,8 @@
 from astroid.builder import parse
 from astroid.util import Uninferable, YES
 
+# TODO
```

Comment:
I've been using the base.NodeNG class for debugging, and the TODO is primarily to remind myself to have a discussion with you about exporting it (we need to) and how we ought to do that.